Bio/SwissProt/__init__.py

```diff
@@ -136,10 +136,18 @@ def read(handle):
         raise ValueError("More than one SwissProt record found")
     return record
 
+def strip_evidence(value):
+    """Strip evidence codes from values.
```

Comment:
The style guidelines in https://www.python.org/dev/peps/pep-0257/ would put a blank line after the one-line description of the function.

Also you could make the examples into a doctest, serving both as documentation and as a test:

```
>>> strip_evidence('ORFNames=N310_11497 {ECO:0000313|EMBL:KFP74115.1};')
'ORFNames=N310_11497;'
>>> strip_evidence('SubName: Full=Tumor protein 63 {ECO:0000313|EMBL:KFP74115.1};')
'SubName: Full=Tumor protein 63;'
```

We can then add ``Bio.Swiss`` to the list of modules with doctests in ``Tests/run_tests.py`` and have this checked automatically.